src/public/app/widgets/dialogs/add_link.js

```diff
@@ -77,30 +77,52 @@ export default class AddLinkDialog extends BasicWidget {
 
                 const linkTitle = this.getLinkType() === 'reference-link' ? null : this.$linkTitle.val();
 
-                this.textTypeWidget.addLink(this.$autoComplete.getSelectedNotePath(), linkTitle);
+                if (this.textTypeWidget) {
```

Comment:
This check appears to be needed because in the context of a script extension theremay not be a textTypeWidget.